rules-tests/PhpSpecToPHPUnit/Rector/Variable/PhpSpecToPHPUnitRector/Fixture/issue_3931.php.inc

```diff
@@ -67,7 +67,7 @@ use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 class PageLinksGeneratorTest extends \PHPUnit\Framework\TestCase
 {
     private \Rector\Tests\PhpSpecToPHPUnit\Rector\Variable\PhpSpecToPHPUnitRector\Fixture\PageLinksGenerator $pageLinksGenerator;
-    private \PHPUnit\Framework\MockObject\MockObject|\Symfony\Bundle\FrameworkBundle\Routing\Router $router;
+    private \PHPUnit\Framework\MockObject\MockObject|\Router $router;
```

Comment:
this is also incorrect for `\Result` as the namespace `Symfony\Bundle\FrameworkBundle\Routing\Router` imported in use statement.